velox/exec/HashPartitionFunction.h

```diff
@@ -58,4 +58,36 @@ class HashPartitionFunction : public core::PartitionFunction {
   SelectivityVector rows_;
   raw_vector<uint64_t> hashes_;
 };
+
+/// Factory class to create HashPartitionFunction
+/// 'keyChannels_' stores the index of keys to partition on, if the key is a
+/// constant, use index 'kConstantChannel' to indicate so and store the constant
+/// value as a base vector in 'constValues_'
+/// The 'constValues_' size is less than or equal to 'keyChannels_' size
```

Comment:
Use public argument names, not private member variables.

keyChannels_ -> keyChannels, etc.